### PR TITLE
HTML minify

### LINKED ISSUE #220: Minify HTML
Minify all HTML output files as part of 11ty build config.

E.g. from [this reference](https://mattclaffey.medium.com/how-to-reduce-you-html-output-in-11ty-ec0c358be2ac), or from one of our other 11ty sites:

```js
const htmlmin = require('html-minifier');
  
 //...

    eleventyConfig.addTransform('htmlmin', (content, outputPath) => {
      if (outputPath.endsWith('.html')) {
        let minified = htmlmin.minify(content, {
          collapseWhitespace: true,
          minifyCSS: true,
          minifyJS: true,
          removeComments: true
        });
        return minified;
      }
      return content;
    });
```

Suggest we perhaps do this for all environments, as opposed to just for `production`, just to get a better steer on any potential pitfalls or minification quirks during development.

This will boost TTFB of responses for the inital load of HTML pages. For example, the homepage should go from 17kb to 14kb. Every lil' helps on that first request.

### PR DESCRIPTION
**Pull request by Pete-Robelou**
- Installed html-minifier package
- Created transforms folder
- Fixed a few syntax issues on /community/team

Closes #220

### CODE REVIEW

File: .eleventy.js [hunk 1]
```diff
@@ -2,11 +2,11 @@ const fs = require('fs');
 
 // Plugins
 const eleventyNavigationPlugin = require('@11ty/eleventy-navigation');
-const sitemap = require('@quasibit/eleventy-plugin-sitemap');
+const build = require('./src/_data/build');
```
Comment (adamduncan) [inline .eleventy.js:5]: Can we omit this if we're not doing env-specific minification?

**Review by adamduncan — APPROVED** [review]
End result here (https://220-minify-html.ceph.io/en/) looks good to me 👍 

If you've not noticed any undesired side-effect, happy to merge

File: .eleventy.js (hunk 1 above)
Comment (Pete-Robelou) [reply .eleventy.js:5]: It's being used for the sitemap plugin.

```
eleventyConfig.addPlugin(sitemap, {
    sitemap: {
      hostname: build.isProduction ? 'https://ceph.io' : 'https://develop.ceph.io',
    },
  });
```

Comment (adamduncan) [reply .eleventy.js:5]: Ah, my bad. Was just confused by the shift in these diffs. Makes sense 👍  Thanks for clarifying

Comment (Pete-Robelou) [reply .eleventy.js:5]: I should probably have mentioned that I sorted the plugins into alphabetical order 🤪

**Pull request merged**